data/CMakeLists.txt

```diff
@@ -16,4 +16,4 @@ install(FILES
   mouse_sloppy
   start
   vars
-  DESTINATION etc/pekwm/)
+  DESTINATION ${SYSCONFDIR}/pekwm/)
```

Comment:
No "/pekwm" here? Your change made me pass

    cmake -DSYSCONFDIR=/etc/pekwm

so here it'll become /etc/pekwm/pekwm, I think

Reply:
The way I understod SYSCONFDIR; I'm suppose to set just the root system-wide config dir, without the subdir, that's suppose to be handled by the project. So I understand your situasjon if one prefixes it with subdir too.
@pekdon You know more about this then me :)

Reply:
Then in the main CMakeLists.txt it should be

    -DSYSCONFDIR="${SYSCONFDIR}/pekwm")

I don't care one way or the other but if the two are not consistent, pekwm can't find system files

Reply:
Yeah I see your point. I'll retest again, think this is a typo again on my behalf when redoing it.